sources/packages/api/src/services/notification/gc-notify.model.ts

```diff
@@ -0,0 +1,35 @@
+export interface GCNotifyResult {
+  content: ContentPayload;
+  id: string;
+  reference: string;
+  scheduled_for: string; //API payload require this naming convention,so we are not following camelcase.
+  template: Template;
+  uri: string;
+}
+
+export interface Template {
+  id: string;
+  uri: string;
+  version: number;
+}
+
+export interface ContentPayload {
+  body: string;
+  from_email: string; //API payload require this naming convention,so we are not following camelcase.
+  subject: number;
+}
+
+export interface RequestPayload {
+  email_address: string; //API payload require this naming convention,so we are not following camelcase.
+  template_id: string; //API payload require this naming convention,so we are not following camelcase.
+  personalisation: Personalisation;
+}
+
+export interface Personalisation {
+  givenNames: string;
+  lastName: string;
+  dob?: string;
```

Comment:
Why dob and dates are optional here?

Reply:
the payload input works even without the DOB